src/apps/allbridge/positions.e2e.ts

```diff
@@ -0,0 +1,22 @@
+import hook from './positions'
+import { NetworkId } from '../../types/networkId'
+
+describe.each([NetworkId['celo-mainnet']])(
+  'getPositionDefinitions for networkId %s',
+  (networkId) => {
+    it('should get the address definitions successfully', async () => {
+      const positions = await hook.getPositionDefinitions(
+        networkId,
+        '0x2b8441ef13333ffa955c9ea5ab5b3692da95260d',
```

Comment:
This wallet address was copied over from the Aave e2e test, but it has not deposited into the Allbridge pool. I'm not sure what address to use here that holds an allbridge position and will continue to as to not break the test?

Reply:
I _think_ this is just a shared wallet where we purchase the required positions. maybe we can buy some allbridge positions here too? cc @jeanregisser

Reply:
I got access from Jean and was able to purchase an Allbridge position so it's working now